ide/coqide/coqOps.ml

```diff
@@ -139,6 +144,7 @@ end = struct
           (List.map (fun (a,b,t) ->
              Printf.sprintf "<%d,%d> %s" a b t) s.tooltips))))
 
+let set_byte_start s v = s.byte_start <- v
```

Comment:
Don't use setters, this is just adding noise. OCaml is not Java.

Reply:
It's not Java?  I had no idea. :-)

I'm not a big fan of setters, but the sentence type was marked private (now removed), so I tried to go with the flow.